python/test/unit/operators/test_blocksparse.py

```diff
@@ -5,6 +5,14 @@
 import triton.ops
 
 
+def is_hip():
```

Comment:
Similarly to #3682, we can use one helper function here.

Reply:
Thanks. changed accordingly.